### ISSUE TITLE
how to encode json to dat

### ISSUE THREAD

**Issue by GuckTubeYT** [GrowtopiaNoobs/GrowtopiaServer#1135]
how? i just decode dat to json, when i want to encode json to dat, but how?

**Comment by WoofHax** [comment]
Open this issues on https://github.com/GrowtopiaNoobs/Growtopia_ItemsDecoder

**Comment by GuckTubeYT** [comment]
ga di jawab

**Comment by DrOreo002** [comment]
> ga di jawab

hadehh bocil gasabaran. Sini w jawab biar engga nangis

**Comment by GuckTubeYT** [comment]
> 
> 
> > ga di jawab
> 
> hadehh bocil gasabaran. Sini w jawab biar engga nangis

nah gitu dong om :)

**Comment by DrOreo002** [comment]
> > > ga di jawab
> > 
> > 
> > hadehh bocil gasabaran. Sini w jawab biar engga nangis
> 
> nah gitu dong om :)

Pinter" dikit yahhh, malu" in ngebacot. Belajar dlu yg bener baru loncat kesini :)

**Comment by GuckTubeYT** [comment]
> 
> 
> > > > ga di jawab
> > > 
> > > 
> > > hadehh bocil gasabaran. Sini w jawab biar engga nangis
> > 
> > 
> > nah gitu dong om :)
> 
> Pinter" dikit yahhh, malu" in ngebacot. Belajar dlu yg bener baru loncat kesini :)
> 
> ENG: I've answered him with Indonesian language because it seems like hes to young to understand proper English :D. But don't worry I don't give him a fake information

ohhhh makasih om udah ngasih tau :V

**Comment by GuckTubeYT** [comment]
blm tau

**Comment by GuckTubeYT** [comment]
pls explain

**Comment by WoofHax** [comment]
//add_item|tile id|display name|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash (0 for none)|0 layer|eCollisionType|HP|Seconds before healing|
//add_clothes|tile id|display name|eItemType|eTileVisualEffect| eTileStorage| frameX| frameY| filename| hash|body part (FACEITEM| PANTS| HAT| etc)|

**Comment by WoofHax** [comment]
add_item|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash (0 for none)|0 layer|eCollisionType|HP|Seconds before healing|

add_clothes|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash|body part (FACEITEM| PANTS| HAT| etc)|

Here some curl if you still don't know then I can't help you anymore

**Comment by DrOreo002** [comment]
> (Custom items block/background)
> add_item|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash (0 for none)|0 layer|eCollisionType|HP|Seconds before healing|
> 
> (custom items clothes)
> add_clothes|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash|body part (FACEITEM| PANTS| HAT| etc)|
> 
> Here some hint for you if you still don't know then I can't help you anymore.

Stop it bro his small brain couldn't handle you massive information

**Comment by WoofHax** [comment]
@DrOreo002 **bruh**, if He still don't know then that I give is really easily example

**Comment by flossboss** [comment]
> (Custom items block/background)
> add_item|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash (0 for none)|0 layer|eCollisionType|HP|Seconds before healing|
> 
> (custom items clothes)
> add_clothes|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash|body part (FACEITEM| PANTS| HAT| etc)|
> 
> Here some hint for you if you still don't know then I can't help you anymore.

How do you use this is what I'm trying to know.

**Comment by DrOreo002** [comment]
> > (Custom items block/background)
> > add_item|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash (0 for none)|0 layer|eCollisionType|HP|Seconds before healing|
> > (custom items clothes)
> > add_clothes|tileid|displayname|eItemType|eTileVisualEffect|eTileStorage| frameX| frameY| filename| hash|body part (FACEITEM| PANTS| HAT| etc)|
> > Here some hint for you if you still don't know then I can't help you anymore.
> 
> How do you use this is what I'm trying to know.

Thats basically the correct item.dat format. You only need to follow that format to create a new item

**Comment by WoofHax** [comment]
![IMG_20200120_124601](https://user-images.githubusercontent.com/52075701/72699339-e317da80-3b82-11ea-866a-0937f5fd6527.jpg)
H0w t0 r34d 1t3ms.d4t

**Comment by GuckTubeYT** [comment]
im still not understand

**Comment by GuckTubeYT** [comment]
can you give me app can encode items dat

**Comment by WoofHax** [comment]
> can you give me app can encode items dat

Kalau lu gk bisa coding gk usah buat 
cpp nya ada disini https://github.com/GrowtopiaNoobs/Growtopia_ItemsDecoder gk ada dimana2

**Comment by GuckTubeYT** [comment]
> 
> 
> > can you give me app can encode items dat
> 
> Kalau lu gk bisa coding gk usah buat
> 
> https://github.com/GrowtopiaNoobs/Growtopia_ItemsDecoder
> ^Decode items.dat cpp Ada disitu

gw udah tau, sisa nya encode

**Comment by DrOreo002** [comment]
> > > can you give me app can encode items dat
> > 
> > 
> > Kalau lu gk bisa coding gk usah buat
> > https://github.com/GrowtopiaNoobs/Growtopia_ItemsDecoder
> > ^Decode items.dat cpp Ada disitu
> 
> gw udah tau, sisa nya encode

Kan dh dibilang klo gabisa gausah buat, kek w dong latihan pertama tau gini 2018 latihan trus smpe karang akhirnya bisa buat bot dan proxy. Bahkan wrapper sendiri 🤔

**Comment by GuckTubeYT** [comment]
ok @DrOreo002 aku harus bersungguh sungguh

**Comment by xXAbood** [comment]
ok nigga and

**Comment by WoofHax** [comment]
This Why I hate Indonesia 
Not you @DrOreo002....

**Comment by JadlionHD** [comment]
> ok @DrOreo002 aku harus bersungguh sungguh

mulai dari dasar dlu biar paham

**Comment by JadlionHD** [comment]
jangan malu-maluin negara kita

**Comment by WoofHax** [comment]
> jangan malu-maluin negara kita

Kamu sama aja ask me how to run gt bot xd

**Comment by JadlionHD** [comment]
> > jangan malu-maluin negara kita
> 
> Lu sama aja ask me how to run gt bot xd
> 
> I don't mean I hate Indo but you guys always ask for code like no using brain, copy paste then compile is make you guys no function and I am not sure why Indo is racist country.
> 
> Even you guys copy paste code gtog.cpp from dirux aka playingo he already help you guys but you guys never credits/thanks to him, so sad to him sending he code for you guys. That why he don't like Indo.
> 
> I very thanks for @NotDark only him I can trust him bcz he can help me everything he can.

yes, but i figured how to compiled by creating issues at gtbot repo to solve my problem

**Comment by GuckTubeYT** [comment]
> 
> 
> jangan malu-maluin negara kita

ok
